latex2sympy.py

```diff
@@ -334,11 +334,13 @@ def convert_comp(comp):
         return convert_matrix(comp.matrix())
     elif comp.func():
         return convert_func(comp.func())
+    elif comp.variable():
```

Comment:
Is it possible for this to be a variable? It looks like variable isn't included as part of `comp` in the `PS` file